Docker.DotNet/Endpoints/IMiscellaneousOperations.cs

```diff
@@ -21,8 +21,14 @@ public interface IMiscellaneousOperations
 
         Task<Stream> GetImageAsTarballAsync(string name, CancellationToken cancellationToken);
 
+        Task<Stream> GetMultipleImagesAsTarballAsync(string[] names, CancellationToken cancellationToken);
+
         Task LoadImageFromTarballAsync(Stream stream, CancellationToken cancellationToken);
 
+        Task<Stream> LoadImageFromTarballWithProgressAsync(Stream stream, CancellationToken cancellationToken);
```

Comment:
I don't think we want to take/produce TarStream. Users will not always want to parse or produce a tar -- sometimes they will just want to pass in a FileStream, for example.